pkg/plugin/resources.go

```diff
@@ -202,7 +210,7 @@ type grafanaOpenAIProxy struct {
 }
 
 func (a *grafanaOpenAIProxy) ServeHTTP(w http.ResponseWriter, req *http.Request) {
-	err := modifyURL(a.settings.LLMGateway.URL+"/openai", req) // GER: FIXME - not durable to / added
```

Comment:
Fixes all look valid to me; I'm just confused by why `+"/openai"` was added in here -- it's not actually used in modifyURL(), right? So I'm guessing this is just an artifact of earlier experimental code

Reply:
Yeah it's been ignored by `modifyURL` for a while I think, looks like an artifact.